app/authorities/sufia/authorities/local/find_works.rb

```diff
@@ -0,0 +1,21 @@
+module Sufia::Authorities
+  class Local::FindWorks < Qa::Authorities::Base
+    attr_reader :q
```

Comment:
@jcoyne Without it, a wrong number of arguments error occurs (given 1, expected 0)

```
 def instance
    klass.new(@subauthority)
```
in questioning_authority/lib/qa/authorities/local/registry.rb.